app/controllers/additional_expenses_controller.rb

```diff
@@ -0,0 +1,32 @@
+class AdditionalExpensesController < ApplicationController
+  def create
+    @additional_expense = AdditionalExpense.new(additional_expense_params)
+    authorize @additional_expense
+
+    respond_to do |format|
+      if @additional_expense.save
+        format.json { render json: @additional_expense.as_json, status: :created }
+      else
+        format.json { render json: @additional_expense.errors.as_json, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  def destroy
+    @additional_expense = AdditionalExpense.find(params[:id])
+    authorize @additional_expense
+
+    @additional_expense.destroy!
+
+    respond_to do |format|
+      format.json { head :no_content }
+    end
```

Comment:
hmm.. rails generated controllers don't worry about it by default... I think the only way a would fail is if there is a  `dependent: :restrict_with_exception` on a model relation, or some bad `dependent: destroy` chain. I think we are safe here since this isn't a 'parent' of any other records.